bitnami/thanos/values-production.yaml

```diff
@@ -176,6 +176,30 @@ querier:
     # periodSeconds: 10
     # successThreshold: 1
     # failureThreshold: 6
+    
+  ## Thanos Querier GRPC TLS parameters
+  ## to configure --grpc-server-tls-cert, --grpc-server-tls-key, --grpc-server-tls-client-ca, --grpc-client-tls-secure, --grpc-client-tls-cert, --grpc-client-tls-key, --grpc-client-tls-ca, --grpc-client-server-name
+  grpcTLS:
+    # tls server side 
```

Comment:
Use `TLS` instead of `tls`, Te same applies to the client comment.
Also, it would be great if you can follow our comments standard for all the comments, that is:
```
      ## <Comment>.
      ## <if applicable> ref: <url>
      ##
```